python/pyspark/sql/functions.py

```diff
@@ -10379,6 +10686,40 @@ def element_at(col: "ColumnOrName", extraction: Any) -> Column:
     return _invoke_function_over_columns("element_at", col, lit(extraction))
 
 
+@try_remote_functions
+def try_element_at(col: "ColumnOrName", extraction: "ColumnOrName") -> Column:
+    """
+    (array, index) - Returns element of array at given (1-based) index. If Index is 0, Spark will
```

Comment:
@panbingkun for `try_element_at`, maybe we should unify the second parameter's name to `index`, in both scala and python

Reply:
Let me fix it in the following PR.

Reply:
@panbingkun I just notice that `try_element_at ` use the same names as `element_at`, so current commit is fine. we don't need any follow up.